.github/ISSUE_TEMPLATE/1-bug.yaml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: Open an issue for a problem you've identified with `fuels`.
+labels: ["triage"]
+body:
+  - type: markdown
```

Comment:
@danielbate! Thanks for getting this moving!

It's great that we're considering improving the GitHub flow further.

However, I'm a big believer in making the right thing the easy thing, and sometimes, the lines between the two can get slightly blurred. This seems right, although not necessarily easy, and I'm hesitant about making this process overly bureaucratic. Perhaps we could benefit from reconsidering the approach with an increased focus on simplicity.

Please let me ponder about this a bit more.

Reply:
Although there are a lot of inputs here, only a few of them are required. But agreed their is a lot more complexity on the page now.

Reply:
I've been thinking if something like this could facilitate the process:
 - https://github.com/FuelLabs/fuels-ts/pull/2802

The issue template could instruct to run `fuels info` and paste its output in the issue form.

Perhaps a `pnpm fuels bug` command could open the new issue form page and fill out most fields.

Reply:
Agreed but this feels like a nice to have. Is the concern that we are asking the user for too much information?

Also potential edge case, but if `fuels info` doesn't run then we have extracted none of the information.

Reply:
Yes, I feel particularly discouraged with opinionated/long issue forms.

Having all the fields seems right, but it'd be awesome if it were easy.

Also looking around at other reps:

 - Rust SDK / blank — https://github.com/FuelLabs/fuels-rs/issues/new
 - Fuel Core / blank — https://github.com/FuelLabs/fuel-core/issues/new
 - Compiler / Offers a blank option — https://github.com/FuelLabs/sway/issues/new/choose

At the very minimum, I think we consider offering a `blank` option.